x-pack/plugins/gis/public/components/layer_panel/view.js

```diff
@@ -60,6 +97,23 @@ export class LayerPanel  extends React.Component {
     );
   }
 
+  _renderSourceProperties() {
+    return this.state.immutableSourceProps.map(({ label, value, link }) => {
+      function renderValue() {
+        if (link) {
+          return (<EuiLink href={link} target="_blank">{value}</EuiLink>);
+        }
+        return (<span>{value}</span>);
+      }
+      return (
+        <p key={label}>
+          <strong className="gisLayerDetails__label">{label}</strong>
```

Comment:
You can remove this class and remove the span around the value since I was trying to do some line-up trickery but it looks like it won't work in most cases including the screenshot you've got. So just let them line-up normally.